cli/loader.go

```diff
@@ -27,6 +27,9 @@ type Loader interface {
 
 	// LoadProfile loads the profile with the given name
 	LoadProfile(ctx context.Context, name string) error
+
+	// ValidateConfig validates the configuration.
+	ValidateConfig(_ context.Context) error
```

Comment:
```suggestion
	ValidateConfig(ctx context.Context) error
```
Should probably stay consistent with the other methods